cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -328,6 +386,11 @@ func createEndpointsLW(kubeClient *kclient.Client) *kcache.ListWatch {
 	return kcache.NewListWatchFromClient(kubeClient, "endpoints", kapi.NamespaceAll, kSelector.Everything())
 }
 
+// Returns a cache.ListWatch that gets all changes to pods.
+func createEndpointsPod(kubeClient *kclient.Client) *kcache.ListWatch {
```

Comment:
should we rename this to createEndpointsPodLW for consistency with similar functions listed above